api/api.proto

```diff
@@ -231,7 +231,7 @@ message StorageResource {
   // True if dedicated as cache
   bool cache = 14;
   // True if the resource is used as thin pool metadata disk
-  bool thin_pool_md = 15;
+  bool pool_metadata_dev = 15;
 }
```

Comment:
You'll need to update the Minor version since we're updating an api.proto field here

Reply:
resolved